Portfolio/src/views/HomeView.vue

```diff
@@ -9,15 +9,15 @@ import { homeViewContent } from "./home-view-content";
     <Wrapper>
         <header class="home-wrapper">
             <div class="home-page-left-container">
-                <h2 class="home-page-small-name">{{ homeViewContent.firstName }}</h2>
-                <h1 class="home-page-title">{{ homeViewContent.titleCareer }}</h1>
+                <h3 class="home-page-small-name">{{ homeViewContent.firstName }}</h3>
+                <h2 class="home-page-title">{{ homeViewContent.titleCareer }}</h2>
             </div>
             <div class="home-page-right-container">
-                <h1 class="home-page-firstName">{{ homeViewContent.firstName }}</h1>
-                <h1 class="home-page-lastName">{{ homeViewContent.lastName }}</h1>
+                <h2 class="home-page-firstName">{{ homeViewContent.firstName }}</h2>
+                <h2 class="home-page-lastName">{{ homeViewContent.lastName }}</h2>
```

Comment:
Should you have multiple heading 2 tags in your page?

Reply:
It's the title and my name, just fixed it.

Reply:
What Jannik means is that u have 2 h2s in a row, which might not make sense. Since your content in this page is just 4 words, first name, last name etc, u might not even need h2 since it is not subsections but just `P `tags for example. What do u think?